src/OrchardCore/OrchardCore.Localization.Core/PortableObject/PortableObjectHtmlLocalizer.cs

```diff
@@ -41,7 +41,7 @@ public override LocalizedHtmlString this[string name]
                 if (_localizer is IPluralStringLocalizer pluralLocalizer && arguments.Length == 1 && arguments[0] is PluralizationArgument)
                 {
                     // Get an unformatted string and all non plural arguments (1st one is the plural count).
-                    var (translation, argumentsWithCount) = pluralLocalizer.GetTranslation(name, arguments);
+                    var (translation, argumentsWithCount) = pluralLocalizer.GetTranslationAsync(name, arguments).GetAwaiter().GetResult();
```

Comment:
All this PR to come back to the same pitfall, we may need to think of another option.

Reply:
Maybe because we didn't do it the right way from the beginning :)